### DISCUSSION TITLE
The land was wrongly sealed #72857

### DISCUSSION THREAD

**Discussion by 72857** [sunflower-land/sunflower-land#1393] [Q&A]
My land ID is 72857, I have only one account, and I have always been playing with a mobile phone.   Today, I was blocked by you if I didn’t log in for five hours.   I have a lot of affection for the game Sunflower, and I started playing it since you first came out.   Yes, I also spent thousands of USDT on this.   I don't want your mistake to block me and I can't play the game.   Please unblock it, thank you!

I made a lot of NFT, I'm very like sunflower NFT, is very valuable, I have been collecting more NFT, I spend a lot of time and money, now you land me a letter, just because I bought gold in the outside, making the golden eggs, I don't think this is wrong, because I paid the money accordingly. This is unfair. I reported the error on Discord, but no one took it seriously. I wanted Adam to help me unblock the land. I'm really sentimental about sunflowers.

Has always been a cell phone address 0 x42878e2738213a156724b7da4108585f24a6b38e 
was playing with my walletHere is my proof of gold purchase

![f1e9eb28be2fa13e739b219366ddb1c](https://user-images.githubusercontent.com/112909630/188520290-7b9e7102-8160-4486-9cfe-855a3f987a46.jpg)

**Comment by Rachboy07** [comment]
Can I help you with something now?Envoyé depuis mon mobile Huawei-------- Message original --------Objet : [sunflower-land/sunflower-land] The land was wrongly sealed #72857 (Discussion #1393)De : 72857 À : sunflower-land/sunflower-land Cc : Subscribed 

My land ID is 72857, I have only one account, and I have always been playing with a mobile phone.   Today, I was blocked by you if I didn’t log in for five hours.   I have a lot of affection for the game Sunflower, and I started playing it since you first came out.   Yes, I also spent thousands of USDT on this.   I don't want your mistake to block me and I can't play the game.   Please unblock it, thank you!

I made a lot of NFT, I'm very like sunflower NFT, is very valuable, I have been collecting more NFT, I spend a lot of time and money, now you land me a letter, just because I bought gold in the outside, making the golden eggs, I don't think this is wrong, because I paid the money accordingly. This is unfair. I reported the error on Discord, but no one took it seriously. I wanted Adam to help me unblock the land. I'm really sentimental about sunflowers.

Has always been a cell phone address 0 x42878e2738213a156724b7da4108585f24a6b38e

was playing with my walletHere is my proof of gold purchase

**Reply by 72857** [reply]
I need the land released

**Comment by adamhannigan** [comment]
Hey @72857 @Rachboy07 Thanks for raising this issue.

Can you clarify what you mean by your farm is sealed?

If the farm was detected by Jigger for possible multiaccounting they require a 3D selfie (no personal details). Are you able to verify your account using the mechanism they provided?

**Reply by 72857** [reply]
Hi Adam! How can I prove that I am the ticket number? I guarantee that I am the ticket number all the time. When I issue the ticket from there, it is deemed that I have purchased the goods, just because I bought the gold outside, I can prove that I bought it, because I paid the same amount of money.

**Reply by 72857** [reply]
Hi Adam, have I checked my situation? I really love Sunflowers, the game is well done, and I put a lot of time and emotion into it myself. I hope you can help me unblock and let me return to Sunflower as soon as possible

**Reply by adamhannigan** [reply]
Hi @72857 

I need more information to help out here. I'm not exactly sure what the premise of this discussion is 😓 

Can you provide exact details of what is happening to your farm? (Farm ID, address, screenshots of game, reproduction steps, etc.)

Thanks 🙏

**Reply by 72857** [reply]
Hello Adam! my farm id 72857,
Wallet address 0x42878e2738213A156724b7Da4108585F24a6b38e
My farm is now banned, I can't log in now
![30686cffc49ae1f1e82bb2b2822981b](https://user-images.githubusercontent.com/112909630/189007663-27409c44-f4dd-4668-a052-c9da486d20e0.jpg)

This is the result of my invoicing. I bought gold from others and made golden eggs. I don't know if this is the case, because I paid the money, and I have evidence to prove that there is a transfer record.

![4d5dcd3cb2316ea86e55b8c712f0520](https://user-images.githubusercontent.com/112909630/189008921-6b43573c-a38a-432c-8c4b-9284777bf7fb.jpg)

![cfd19089b94a2a49657f612e6df8f59](https://user-images.githubusercontent.com/112909630/189008960-7f5cb3ce-7b93-4b81-97d0-53b849440973.jpg)

Finally, I hope Adam can help me unblock it, and I also hope that Sunflower will do better and better. I will continue to support it so that more people will discover Sunflower.

**Reply by 72857** [reply]
ianwang#7604 There is also a person in our Chinese community who bought the resources of the script number, but you have given the fastest unblocking, when can you unblock it for me? It's the golden egg

**Reply by 72857** [reply]
Adam help to unblock it, I know what to do every day without sunflowers! Sunflowers are already a part of my life, without sunflowers I don't know what's the point of life

**Reply by 72857** [reply]
> 你好@72857
> 
> 我需要更多信息来帮助解决这个问题。我不确定这次讨论的前提是什么😓
> 
> 您能否提供您农场发生的事情的确切细节？（农场ID、地址、游戏截图、复制步骤等）
> 
> 谢谢🙏

**Reply by 72857** [reply]
> 你好@72857
> 
> 我需要更多信息来帮助解决这个问题。我不确定这次讨论的前提是什么😓
> 
> 您能否提供您农场发生的事情的确切细节？（农场ID、地址、游戏截图、复制步骤等）
> 
> 谢谢🙏

My land ID is 72857, I only have one account, and I have been playing with my mobile phone all the time. It was blocked in September. I have been paying attention to the development of Sunflower. I like this game very much. I also called many friends to play together. But my friends are playing, and my land is gone. I have a lot of affection for the game Sunflower. From you Just started playing when it came out. Yep, I've spent thousands of dollars on this too. I can't play the game either, please unblock, thank you! I can assure you that I only have one account, because I mistakenly bought gold from others in the group, you think I have multiple accounts, now you can check, I really only have one account to play, and I am very Love this game. Please help thank you, you are the best, sunflower is the best


Land ID 72857
My address 0x42878e2738213A156724b7Da4108585F24a6b38e

**Comment by Python-DW** [comment]
> My land ID is 72857, I have only one account, and I have always been playing with a mobile phone. Today, I was blocked by you if I didn’t log in for five hours. I have a lot of affection for the game Sunflower, and I started playing it since you first came out. Yes, I also spent thousands of USDT on this. I don't want your mistake to block me and I can't play the game. Please unblock it, thank you!
> 
> I made a lot of NFT, I'm very like sunflower NFT, is very valuable, I have been collecting more NFT, I spend a lot of time and money, now you land me a letter, just because I bought gold in the outside, making the golden eggs, I don't think this is wrong, because I paid the money accordingly. This is unfair. I reported the error on Discord, but no one took it seriously. I wanted Adam to help me unblock the land. I'm really sentimental about sunflowers.
> 
> Has always been a cell phone address 0 x42878e2738213a156724b7da4108585f24a6b38e was playing with my walletHere is my proof of gold purchase
> 
> ![f1e9eb28be2fa13e739b219366ddb1c](https://user-images.githubusercontent.com/112909630/188520290-7b9e7102-8160-4486-9cfe-855a3f987a46.jpg)

On buying gold to do NFT, I do not think there is any problem, even if not recognized this way, direct closure, punishment is too serious
